.pep8

```diff
@@ -0,0 +1,6 @@
+[pep8]
+max-line-length=256
+
+[flake8]
+max-line-length=256
+ignore=E731
```

Comment:
By the way, do you use lambdas that much that you need this?
I haven't seen any lambda assignments so far. Though I also use lambdas, they are reduced to value processing one-liners in arguments parser and in array to string conversions.
Even if so, you better use ` some= lambda x: assignment(x)  # NOQA: E731` instead, and add a case to mark all NOQA's in the diff, and mark the commit with these diffs for manual review.